recipe/migrations/boost172.yaml

```diff
@@ -3,11 +3,12 @@ __migrator:
   kind:
     version
   migration_number:
-    2
+    3
```

Comment:
Changing this would make conda-smithy remove 1.72 until the new migration is hit right?

Reply:
The new migration would startup almost immediately and 172 builds already exist, so the initial ones should go fast. However, yes, AFAIK, you are correct.

Reply:
Wouldn't it be better to make a new 1.70_and_1.72 migrator and stop the 1.72 migrator manually without removing the yaml?

Reply:
Doing that is equivalent to change the pin on latest in this PR and some of the data in this file, right?

Reply:
What do you mean by changing some of the data in this file?

Reply:
Ohhhh just resetting the timestamp and renaming the file etc.

Reply:
No, if we do that, the feedstocks that have already been migrated will revert to 1.70 until the bot sends a PR with the new migration.

Reply:
We change the value in conda_build_config.yaml to 1.72 as well. 

This is what I mean by "change the pin on latest". 

Then feedstocks won't get reverted.

Reply:
Then feedstocks that haven't been migrated starts failing, right?

Reply:
What do you mean?

Reply:
We'd have the 1.70 migration going at the same time, backporting all of the pins we need.

Reply:
Feedstocks that haven't been migrated to 1.72 yet will start getting 1.72 before the bot has migrated their upstreams right?

Reply:
Let's just add another 1.70 and 1.72 migrator and try setting pause on the 1.72 one. Hold on.

Reply:
ok @isuruf - check out the latest version - the paused key stops the migration without removing currently migrated things.